py3status/module.py

```diff
@@ -635,6 +642,13 @@ def deprecation_log(item):
                 param = True
             self.allow_urgent = param
 
+            # format_urgent: get the param from the config if supplied.
+            fn = self._py3_wrapper.get_config_attribute
+            param = fn(self.module_full_name, 'format_urgent')
+            if hasattr(param, 'none_setting'):
+                param = '{format}'
```

Comment:
This is inefficient because we would be reformatting to get the same output.

If there is none then you should do no work